requirements.txt

```diff
@@ -1,7 +1,6 @@
 bsdiff4==1.1.4
 cx_Freeze==4.3.4
 enum34
-lupa
```

Comment:
Why have you removed this dependency? It's a runtime dependency of faftools and we're using it to parse lua code.

You left no comment about this in the PR, either.